src/docs/dto/reservation.dto.ts

```
@@ -0,0 +1,918 @@
+import { BasePaginationQueryDto } from '@/common/dto/request';
+import { PaginationMetadata } from '@/common/dto/response';
+import { ReservationCategory } from '@/common/enums/reservation-category';
+import { UserReservationStatus } from '@/common/enums/user-reservation-status';
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
```

ESLint

[error] 'Type' is defined but never used.

(@typescript-eslint/no-unused-vars)